test/e2e_test.ts

```diff
@@ -49,5 +49,5 @@ describe('golden file tests', () => {
     goldenJs.push('test_files/import_from_goog/closure_OtherModule.js');
     const externs = tests.map(t => t.externsPath).filter(fs.existsSync);
     checkClosureCompile(goldenJs, externs, done);
-  });
+  }, 10000 /* ms timeout */);
```

Comment:
should this interact with process.env['TEST_TIMEOUT'] in some way? do we really care to have jasmine enforce timeouts as well as bazel?

Reply:
So `small` tests have a 60 second timeout in bazel (go figure...). https://docs.bazel.build/versions/master/test-encyclopedia.html

That seems a bit excessive, and also we don't have a guarantee that this is the only test executing in its suite. So I'd rather keep it tight here and relevant to this local test, not deducted from some outer limitation.

Reply:
If this is the only test where we set the timeout, then for other tests we rely on `jasmine.DEFAULT_TIMEOUT_INTERVAL` being the shorter value you like.

In https://github.com/bazelbuild/rules_nodejs/pull/53 I propose to make the default timeout longer, so now all tests other than this one would get 5m (since they are timeout="moderate")

Maybe this is all too nit-picky.